W-19128919 agent fabric exchange GA - jh (#319)
mulesoft/docs-exchange: modules/ROOT/pages/to-request-access.adoc

A client application can access an API either through a contract with only that API or through a contract with one or more API Groups that include the API. In this latter case, the least restrictive available contract for the API is applied.

== Agent, LLM, and MCP Server Access
In Exchange, agents, LLMs, and MCP servers are assets with a portal. The *Instances* page on the portal lists all of the asset's instances.

When a developer requests access to these assets for a client application, the developer selects an instance, selects a client application, and selects a service-level agreement (SLA) if available.

. Select the asset from the Exchange catalog to view the details page.
. Click *Request access*.
. Choose the instance from the list.
. If you are using an existing application, choose from the list of applications.
+
An application and its instance use the same client provider. If an organization administrator has configured xref:api-manager::configure-multiple-credential-providers.adoc[multiple client providers] in Anypoint API Manager, then the application list shows only the applications that use the same client provider as the selected instance.

> . If you do not have an application to which to provide access, click *Select application > Create a new application*. 

Reviewer: ```suggestion
. If you don't have an application to provide access to, select *Select application > Create a new application*. 
```
